programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1634,6 +1634,34 @@ static void run(int argc, char *argv[]) {
     signal(SIGINT, interrupt_handler);
 #endif
 
+#if _WIN32
```

Comment:
nit: there's two #if _WIN32 in a row. seems like this and line 1636 could be removed too, right?

Reply:
yes, the ifdefs can be combined. I reordered this section so the logger was initialized sooner and didn't pay attention to the preceding ifdef at all.

eventually, hopefully soon, all the the OS-specific ifdefs will be removed from ziti-edge-tunnel.